Taskfile.yml

```diff
@@ -146,6 +146,13 @@ tasks:
       - bin/goimports{{exeExt}} -format-only -local=github.com/FerretDB/FerretDB -w .
       - bin/gofumpt{{exeExt}} -w .
 
+  fmt-yaml:
+    desc: "Format yaml"
+    cmds:
+      - find . -name "*.yml" -exec chmod o+w {} \;
+      - docker run --rm -v $(pwd):/work tmknom/prettier --write --parser=yaml "./**/*.yml"
```

Comment:
I like how this formatter works! And I like that you added a docker-compose configuration for it, so it can be used the same way how other tasks in this file.

For this command itself, let's also use `docker compose` with the configuration you provided, not just `docker`?

Reply:
updated to use docker compose.